modules/amp-core/pages/index.adoc

```diff
@@ -20,7 +20,7 @@ This tutorial shows:
 
 == 1. Getting AMP CAST
 
-The customer's premium license includes AMP CAST. To obtain a self-hosted package, reach https://ampsupport.atlassian.net/servicedesk/customer/portal/6[AMP Support].
+The customer's premier license includes AMP CAST. To obtain a self-hosted package, reach https://ampsupport.atlassian.net/servicedesk/customer/portal/6[AMP Support].
```

Comment:
@mchacong  the link is incorrect. it does not resolve.
https://ampsupport.atlassian.net/servicedesk/customer/portal/6

But we can use the official redirect instead: https://player.akamai.com/support

Reply:
changed